2015/presentations/latajacy_waz_czyli_szybkie_prototypowanie_sprzetu_przy_pomocy_pythona/text.md

```diff
@@ -0,0 +1,105 @@
+# Latający wąż, czyli szybkie prototypowanie sprzętu przy pomocy Pythona - Damian Gadomski
+
+Artykuł ten dotyczy prototypowania sprzętu na platformie MicroPython. W pierwszej części omówiona została implementacja języka na mikrokontolery z rodziny ARM, dostępne biblioteki oraz pokrewny projekt pyBoard. Dokonano porównania wydajności MicroPythona względem innych popularnych platform, a także opisano budowę i zasadę działania dronów latających w zakresie umożliwiającym zrozumienie problemów, przed którymi staje sterownik lotu takiego urządzenia.
+
+## Wstęp
+
+Czasami zdarzają się projekty bardziej skomplikowane niż inne. Takie, gdzie nie do końca wiadomo w jaki sposób osiągnąć oczekiwany rezultat. Takie, gdzie potrzeba trochę pracy badawczej i testów, bo zagadnienie nad którym pracujemy nie ma oczywistego rozwiązania. Często też nie wiadomo jaki rezultat będzie najbardziej użyteczny i praktyczny. Chcielibyśmy wtedy mieć możliwość zrobienia prototypu, który stale będziemy ulepszać. Dobrze jeśli kolejne zmiany nie będą obarczone dużym kosztem, ich wprowadzenie będzie szybkie. Może nawet chcielibyśmy przetestować kilka algorytmów jednocześnie?
+
+### Inteligenta łazienka
+
+Nie musi to być od razu ogromny system, po prostu, niewielki problem do rozwiązania. Przykładem może być kwestia włączania ogrzewania i wyciągu (wentylatora) w inteligentej łazience. Po wzięciu prysznica chcielibyśmy, żeby ręcznik odwieszony na grzejnik jak najszybciej wysechł, a wentylator jak najszybciej pozbył się wilgoci z łazienki. Wbrew pozorom nie jest to takie banalne.  W jaki sposób wykryć, że ktoś wziął prysznic? Odpowiednio długo zapalone światło, podwyższona wilgotność w łazience? Jaka zmiana wilgotności świadczy o wziętym prysznicu? A może właśnie zostało rozwieszone pranie? Jak długo grzejnik powinien być włączony aby ręcznik wysechł? Jak długo powinien działać wentylator? Powinien uruchomić się od razu, czy po wyjściu z łazienki?
```

Comment:
Zmieniłbym pierwsze zdanie na:

*Nie musi to być od razu ogromny system, lecz po prostu, niewielki problem do rozwiązania.*

Oprócz tego:

inteligentej -> inteligent**n**ej